### PR TITLE
Update membership

### PR DESCRIPTION
**Pull request by ehsteve**

### CODE REVIEW

**Comment by ehsteve** [comment]
@dpshelio since you are the vice-chair please take a look and verify this change. Thanks!

**Comment by Cadair** [comment]
While I realise you have put in the two year terms, do we want to curtail them so we always have board elections in December? This would be good for a community point of view I think.

**Comment by ehsteve** [comment]
While, in theory, I agree, in practice it seems like Dec elections are a bad idea because we obviously let them pass without thinking about it. I'd rather we move all elections to March.

**Comment by Cadair** [comment]
well that's fine to, as long as we shuffle them that way. I think there is something in trying to do it all at the same time every year, so perhaps we can have the next set of elections in March 2018?

File: SEP-0006.md [hunk 1]
```diff
@@ -25,23 +25,23 @@ The SunPy board is composed of the following members (in alphabetical order)
...
-| Thomas Robitaille             |                   | 16 Apr 2014 | 31 Dec 2017 |
+| Sabrina Savage                |                   | 14 Mar 2017 | 14 Mar 2019 |
 | Albert Shih                   |                   | 17 Mar 2014 | 31 Dec 2018 |
 
 # Executive Director
 The executive director (also referred to as the lead developer) is
 
 | Name           | Start        | Expiration |
 |----------------|--------------|------------|
-| Stuart Mumford | 23 Sep 2015  | 23 Sep 2016|
```
Comment (rhewett) [inline SEP-0006.md]: Should keep the history of old executive directors?  Can be updated later.

**Pull request merged**